pandas/tests/extension/arrow/test_bool.py

```diff
@@ -39,6 +39,14 @@ class TestInterface(BaseArrowTests, base.BaseInterfaceTests):
     def test_repr(self, data):
         raise pytest.skip("TODO")
 
+    def test_shift_non_empty_array(self, data):
+        raise pytest.skip("Skipping because the implementation of"
```

Comment:
is there an issue for this?

Reply:
I'll check through the issues list. But the reason I added this is @ TomAugspurger's recommendation above.

> Feel free to add skips for the arrow tests. That implementation is pretty buggy right now.

Reply:
can you add an issue for this